### PR TITLE
TW-1109: Add separated event to kord.fi pool

### PR DESCRIPTION
**Pull request by svjatoslavtt**

### CODE REVIEW

**Review by keshan3262 — COMMENTED** [review]
See comments from @alex-tsx

File: src/modals/manage-earn-opportunity-modal/index.tsx [hunk 1]
```diff
@@ -187,18 +197,27 @@ export const ManageEarnOpportunityModal: FC = () => {
             disabled={pageIsLoading || !earnOpportunityIsSupported || stakeFormErrorsVisible || stakeFormSubmitting}
             onPress={handleDepositClick}
             testID={ManageEarnOpportunityModalSelectors.depositButton}
+            testIDProperties={{
+              ...(!stakeFormErrorsPresent && {
+                name: stakedTokenData.value.asset.symbol,
+                value: +stakedTokenData.value.amount
```
Comment (lendihop) [inline src/modals/manage-earn-opportunity-modal/index.tsx:203]: pls use `Number()` here

**Review by lendihop — APPROVED** [review]

**Review by lendihop — APPROVED** [review]

File: src/modals/manage-earn-opportunity-modal/stake-form/use-stake-formik.ts [hunk 2]
```diff
@@ -76,11 +76,14 @@ export const useStakeFormik = (earnOpportunity?: EarnOpportunity, stake?: UserSt
         return;
       }
 
-      trackEvent('STAKE_FORM_SUBMIT', AnalyticsEventCategory.FormSubmit, {
-        farmAddress: earnOpportunity.contractAddress,
-        token: asset.symbol,
-        atomicAmount: amount.toFixed()
-      });
+      // TODO: disable this event for all earn opportunities at the next tasks
```
[the remark this answers is not in the document]
Comment (keshan3262) [reply src/modals/manage-earn-opportunity-modal/stake-form/use-stake-formik.ts:79]: This comment in code is irrelevant and will be removed. There is only a task for removing it for Youves savings (https://madfish.atlassian.net/browse/TW-1117) for now

File: src/modals/manage-earn-opportunity-modal/withdraw-form/use-withdraw-formik.ts [hunk 3]
```diff
@@ -83,10 +83,13 @@ export const useWithdrawFormik = (earnOpportunity?: EarnOpportunity, stake?: Use
 
       const { tokenOption, amountOptionIndex } = values;
       const { token } = tokenOption;
-      trackEvent('WITHDRAW_FORM_SUBMIT', AnalyticsEventCategory.FormSubmit, {
-        farmAddress: earnOpportunity.contractAddress,
-        token: token.symbol
-      });
+      // TODO: disable this event for all earn opportunities at the next tasks
```
[the remark this answers is not in the document]
Comment (keshan3262) [reply src/modals/manage-earn-opportunity-modal/withdraw-form/use-withdraw-formik.ts:86]: https://github.com/madfish-solutions/templewallet-mobile/pull/958/files#r1345728147

**Review by lendihop — APPROVED** [review]

**Review by lendihop — APPROVED** [review]

File: src/modals/manage-earn-opportunity-modal/index.tsx [hunk 4]
```diff
@@ -173,15 +173,15 @@ export const ManageEarnOpportunityModal: FC = () => {
 
     const { tokenOption, amountOptionIndex } = withdrawFormValues;
     const percentage = PERCENTAGE_OPTIONS[amountOptionIndex];
-    const atomicAmount = new BigNumber(stake.depositAmountAtomic ?? 0)
+    const atomicAmount = new BigNumber(stake?.depositAmountAtomic ?? 0)
```
[the remark this answers is not in the document]
Comment (keshan3262) [reply src/modals/manage-earn-opportunity-modal/index.tsx:176]: Yes, you're right. I'm afraid `"noUncheckedIndexedAccess": true,` in tsconfig.json can help us but adding it there is too expensive

**Review by lendihop — APPROVED** [review]

**Review by lendihop — APPROVED** [review]

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.16_965

**Pull request merged**